client/app/views/steps/subviews/progression.coffee

```diff
@@ -0,0 +1,4 @@
+{LayoutView} = require 'backbone.marionette'
+
+module.exports = class ProgressionView extends LayoutView
```

Comment:
Ok, actually it's not working with a View, but it does with an ItemView. What do you think is better ?